packages/forma-36-react-components/src/components/IconButton/IconButton.tsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import cn from 'classnames';
-import Icon, { IconProps } from '../Icon';
-import TabFocusTrap from '../TabFocusTrap';
+import { IconProps } from '../Icon';
+import Button from '../Button';
 
 import styles from './IconButton.css';
 
 export interface IconButtonProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * It controls the size button
+   * 'trimmed' to be removed in v4, don't use
+   */
+  size?: 'small' | 'large' | 'trimmed';
```

Comment:
@mshaaban0 I'm wondering, for the future if we should come up with the same sizes shared between Icon and IconButton.

Reply:
maybe, as per design the icon size is the same, the button size changes.
But for the size naming, I agree we should align them yes.

Reply:
Exactly, cool cool cool.